src/concrete/ml/quantization/quantized_module.py

```diff
@@ -90,13 +92,15 @@ class QuantizedModule:
     input_quantizers: List[UniformQuantizer]
     output_quantizers: List[UniformQuantizer]
     fhe_circuit: Union[None, Circuit]
+    _preprocessing_module: Optional[NumpyModule] = None
```

Comment:
to keep consistency we should remove this + add the `Optional[NumpyModule]` typing to when we init the attribute

Reply:
I am not sure I follow the rules for attributes why would preprocessing_module not belong here vs the above attributes?

Reply:
actually these attribute typings are not necessary and should be removed as they all are defined in the init (some are missing some typings but we should add the)

normally these class attribute typings are defined here when not done in the init (in case they are instantiated in subclasses' init for example, like done in built-in models), so it's an error to have them here

as for `_preprocessing_module` specifically, it does not make sense to put a default value if not relevant. And imo it is not relevant here because we instantiate it in the init anyway. So it would be preferable to remove it entirely. And I'll open a follow-up PR to remove the other ones !